Documentation/Upgrade/rook-upgrade.md

```diff
@@ -49,16 +50,18 @@ those releases.
 ## Breaking changes in v1.13
 
 * The minimum supported version of Kubernetes is v1.23.
-* Support for the admission controller/webhooks has been removed. If admission controller/webhooks is enabled, disable by changing
-`ROOK_DISABLE_ADMISSION_CONTROLLER: "true"` in operator.yaml before upgrading to rook v1.13. CRD validation is now enabled with [Common Expression Language](https://kubernetes.io/docs/reference/using-api/cel/). This requires Kubernetes version 1.25 or higher.
-
-## Breaking changes in v1.12
-
-* The minimum supported version of Kubernetes is v1.22.
+  Upgrade to Kubernetes v1.23 or higher before upgrading Rook.
+* The minimum supported version of Ceph is v17.2.0.
+  If a lower version is currently deployed, [Upgrade Ceph](./ceph-upgrade.md) before upgrading Rook.
 * CephCSI CephFS driver introduced a breaking change in v3.9.0. If any existing CephFS storageclass in
   the cluster has `MountOptions` parameter set, follow the steps mentioned in the
   [CephCSI upgrade guide](https://github.com/ceph/ceph-csi/blob/v3.9.0/docs/ceph-csi-upgrade.md/#upgrading-cephfs)
   to ensure a smooth upgrade.
+* Support for the admission controller has been removed. CRD validation is now enabled with
+  [Validating Admission Policies](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/).
```

Comment:
The validating admission policies say's beta is available from 1.28 but the CEL is in beta since 1.25 a
```
Beta since Kubernetes 1.25, CEL Validation Rules allow CRD authors to express validation constraints on their fields using a rich expression language, [CEL](https://github.com/google/cel-spec). This article explores how you can use validation rules to implement a few common immutability patterns directly in the manifest for a CRD.
```

Reply:
I've been confused by the way k8s does those notes as well. The current version of k8s is 1.28, where the feature is beta, and that is what the note means. The note isn't intended to indicate the release where the feature entered beta.

Reply:
Also, we should continue to refer to this by the k8s feature name, which is "Validating Admission Policy." CEL is the language used to specify VAPs, but it is not the feature name. I also don't think that it's important for Rook users to know what CEL is since they won't be writing the rules for it. It's just an implementation detail, and it seems better to not overcomplicate things for them.